hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -471,6 +473,29 @@ public HollowConsumerMetrics getMetrics() {
             return null;
         }
 
+        HollowConsumer.HeaderBlob retrieveHeaderBlob(long desiredVersion);
```

Comment:
We need a default implementation here too

Reply:
default == `throw new UnsupportedOperationException()`?

New features like this should be entirely optional.